docs/signed-apk-android.md

```diff
@@ -7,14 +7,20 @@ Android requires that all apps be digitally signed with a certificate before the
 
 ## Generating an upload key
 
-You can generate a private signing key using `keytool`. On Windows `keytool` must be run from `C:\Program Files\Java\jdkx.x.x_x\bin`.
+You can generate a private signing key using `keytool`.
+
+### Windows
+
+On Windows `keytool` must be run from `C:\Program Files\Java\jdkx.x.x_x\bin`.
 
     keytool -genkeypair -v -storetype PKCS12 -keystore my-upload-key.keystore -alias my-key-alias -keyalg RSA -keysize 2048 -validity 10000
 
 This command prompts you for passwords for the keystore and key and for the Distinguished Name fields for your key. It then generates the keystore as a file called `my-upload-key.keystore`.
 
 The keystore contains a single key, valid for 10000 days. The alias is a name that you will use later when signing your app, so remember to take note of the alias.
 
+### Mac
+
 On Mac, if you're not sure where your JDK bin folder is, then perform the following command to find it:
```

Comment:
```suggestion
### macOS

On macOS, if you're not sure where your JDK bin folder is, then perform the following command to find it:
```

Let's fix the naming since we are changing those lines already.